connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaStatusBackingStore.java

```diff
@@ -137,11 +137,6 @@
     private int generation;
     private SharedTopicAdmin ownTopicAdmin;
 
-    @Deprecated
```

Comment:
Do you know why we had this deprecated? Do we consider it a public API?

Below we have code which checks for null `SharedTopicAdmin`. Seems like we can probably get rid of that. The only usage I could see was in a test case.

Reply:
It's not public API but looking at the history it looks like we're aware of external users and we decided to keep compatibility. For that reason, I've reverted this change.